br_nfe/models/invoice_eletronic.py

```diff
@@ -374,8 +383,8 @@ def _prepare_eletronic_invoice_item(self, item, invoice):
         }
         if item.tipo_produto == 'service':
             retencoes = item.pis_valor_retencao + \
-                item.cofins_valor_retencao + item.inss_valor_retencao + \
-                item.irrf_valor_retencao + item.csll_valor_retencao
+                        item.cofins_valor_retencao + item.inss_valor_retencao + \
```

Comment:
Identação em excesso:
"br_nfe/models/invoice_eletronic.py:386:25: E126 continuation line over-indented for hanging indent"